recipes/stlearn/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "stlearn" %}
+{% set version = "0.4.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stlearn-{{ version }}.tar.gz
+  sha256: fd861dc94bf77b1acf744c499cefdf41f9c59ec548810b3e256a88c68a3483aa
+
+build:
+  entry_points:
+    - stlearn=stlearn.app.app:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - bokeh >=2.4.2
+    - leidenalg
+    - louvain
+    - numpy >=1.18,<1.22
+    - pillow >=9.0.1
+    - python >=3.7
+    - scanpy >=1.8.2
+    - scikit-image >=0.19.2
+    - tensorflow
+    - xlrd <2.0
+
+test:
+  imports:
+    - stlearn
+  commands:
+    - pip check
```

```suggestion
    - pip check
    - stlearn --help
```
Just to test the entry points